deployment/ansible/roles/model-my-watershed.celery-worker/meta/main.yml

```diff
@@ -5,3 +5,4 @@ dependencies:
   - { role: "azavea.pip" }
   - { role: "model-my-watershed.monitoring", collectd_prefix: "collectd.worker.", when: "['test'] | is_not_in(group_names)" }
   - { role: "model-my-watershed.celery" }
+  - { role: "model-my-watershed.rwd" }
```

Comment:
I made this a dependency of the celery-worker role because the celery tasks which spawn RWD requests depend on environment variables created by the rwd role.